typescript-generator-gradle-plugin/src/main/java/cz/habarta/typescript/generator/gradle/GenerateTask.java

```diff
@@ -134,13 +134,12 @@ public void generate() throws Exception {
         // class loader
         final Set<URL> urls = new LinkedHashSet<>();
         for (Task task : getProject().getTasks()) {
-            if (task.getName().startsWith("compile") && !task.getName().startsWith("compileTest")) {
+            if ("compileClasspath".equals(task.getName())) {
```

Comment:
This need to stay because otherwise classpath would not contain compiled classes.